mock/src/main/java/jinsist/mock/MethodToStubNotFound.java

```diff
@@ -1,6 +1,8 @@
 package jinsist.mock;
 
 class MethodToStubNotFound extends RuntimeException {
+    private static final long serialVersionUID = 1L;
```

Comment:
Where do you see a warning for this? I cannot find it in gradle log. Looks like not needed for this project - we do not use different versions at the same time.